Patch for Debian bug #556203

pam_modutil.h should include sys/types.h, so that it stands alone without
callers having to do additional busywork.

Authors: Russ Allbery <rra@debian.org>

Upstream status: not yet submitted

Index: pam.debian/libpam/include/security/pam_modutil.h
===================================================================
--- pam.debian.orig/libpam/include/security/pam_modutil.h
+++ pam.debian/libpam/include/security/pam_modutil.h
@@ -51,6 +51,7 @@
 extern "C" {
 #endif
 
+#include <sys/types.h>
 #include <security/_pam_types.h>
 
 extern struct passwd * PAM_NONNULL((1,2))
